parcel_test.go

```diff
@@ -31,58 +30,175 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		t.Fatalf("failed to open database: %v", err)
+	}
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	id, err := store.Add(parcel)
+	if assert.NoError(t, err, "Add() should not return error") {
```

Comment:
Не нужно усложнять логику, просто проверь, есть ли ошибка или нет.
Если ошибка, то мы должны завершить тестирование, поэтому используем обязательную проверку.
```suggestion
	require.NoError(t, err, "Add() should not return error")
```